superset-frontend/src/SqlLab/components/SouthPane.jsx

```diff
@@ -59,6 +60,29 @@ const defaultProps = {
   offline: false,
 };
 
+const StyledPane = styled.div`
+  width: 100%;
+
+  .ant-tabs .ant-tabs-content-holder {
+    overflow: visible;
+  }
+
+  .SouthPaneTabs {
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+  }
+  .tab-content {
+    .alert {
+      margin-top: 10px;
+    }
+
+    button.fetch {
+      margin-top: 10px;
```

Comment:
```suggestion
      margin-top: ${({ theme }) => theme.gridUnit * 2}px;
```
Same as above... just setting ourselves up with the gridUnit. Then one day we (or users!) can make the gridUnit 5px if they want things aired out, or 3px if they want it tightened up. And... it matches Figma ;)